docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -93,6 +93,25 @@ cube(`Orders`, {
 });
 ```
 
+## Rollup Only Mode
```

Comment:
Maybe we need to write an additional notice: Please don't combine CUBEJS_ROLLUP_ONLY & CUBEJS_REFRESH_WORKER for one node, because it will not work (cube.js disables rollupOnlyMode when refresh mode is enabled) + similar thing will be for devMode = true